src/react/components/pages/editorPage/canvas.test.tsx

```diff
@@ -92,36 +95,50 @@ describe("Editor Canvas", () => {
     });
 
     it("canvas is updated when asset loads", () => {
+        const wrapper = createComponent();
         wrapper.find(AssetPreview).props().onLoaded(expect.any(HTMLImageElement));
 
         expect(wrapper.instance().editor.addContentSource).toBeCalledWith(expect.any(HTMLImageElement));
         expect(wrapper.state().contentSource).toEqual(expect.any(HTMLImageElement));
     });
 
     it("canvas is enabled when an asset is deactivated", () => {
+        const wrapper = createComponent();
         wrapper.find(AssetPreview).props().onDeactivated(expect.any(HTMLImageElement));
 
         expect(wrapper.instance().editor.addContentSource).toBeCalledWith(expect.any(HTMLImageElement));
         expect(wrapper.state().canvasEnabled).toEqual(true);
     });
 
     it("canvas is deactivated when an asset is activated", () => {
+        const wrapper = createComponent();
```

Comment:
Can you call this line `const wrapper = createComponent();` in `beforeEach()`?

Reply:
I was doing that, but it had introduced a few race conditions in the expectations for each test